osu.Game/Overlays/Comments/DrawableComment.cs

```diff
@@ -403,6 +409,28 @@ private void deleteCommentRequest()
             api.Queue(request);
         }
 
+        public void ReportComment(CommentReportReason reason, string comment)
+        {
+            actionsContainer.Hide();
+            actionsLoading.Show();
+            var request = new CommentReportRequest(Comment.Id, reason, comment);
+            request.Success += () => Schedule(() =>
+            {
+                actionsLoading.Hide();
+                notificationOverlay?.Post(new SimpleNotification
+                {
+                    Icon = FontAwesome.Solid.CheckCircle,
+                    Text = "The comment reported successfully."
+                });
```

Comment:
Not sure this should exist, but an OSD similar to the one in `ExternalLinkButton` may be better than a full notification.